blog/node-22-ecosystem-update.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Moving our Ecosystem to Node 22
+date: 2024-12-02T00:00:00.000Z
+authors: VerteDinde
+slug: ecosystem-node-22
+tags: [community, ecosystem]
+---
+
+In early 2025, Electron’s npm ecosystem repos (under the `@electron/` and `@electron-forge/` namespaces) will move to Node.js 22 as the minimum supported version.
+
+---
+
+### What does this mean?
+
+In the past, packages in Electron’s npm ecosystem (Forge, Packager, etc) have supported Node versions for as long as possible, even after a version has reached its End-Of-Life (EOL) date. This is done to make sure we don’t fragment the ecosystem—we understand that many projects depend on older versions of Node, and we don’t want to risk stranding those projects unless there was a pressing reason to upgrade.
+
+Over time, maintenance of this policy has become increasingly difficult for a few reasons:
```

Comment:
This phrasing "maintenance of this policy" is a little confusing about whether the _version_ is changing (a one-time bump) or whether the _policy_ is changing too (such as, more frequent bumps in the future)

If it's the latter, then we should be a lot more explicit about it here. If it's the former, I'd rephrase:

```suggestion
Over time, using Node.js 14 as our minimum version has become increasingly difficult for a few reasons:
```